functions/linkedInWeeklyCsvExport.js

```diff
@@ -0,0 +1,53 @@
+const AWS = require('aws-sdk');
+const CSV = require('csv');
+const moment = require('moment');
+
+const DynamoDB = new AWS.DynamoDB.DocumentClient();
+const S3 = new AWS.S3();
+
+const scan = params => new Promise(
+  (resolve, reject) => DynamoDB.scan(params, (err, data) => err ? reject(err) : resolve(data))
+);
+
+const stringify = params => new Promise(
+  (resolve, reject) => CSV.stringify(params, { header: true }, (err, data) => err ? reject(err) : resolve(data))
+);
+
+const putObject = params => new Promise(
+  (resolve, reject) => S3.putObject(params, (err, data) => err ? reject(err) : resolve(data))
+);
+
+const dumpTables = async (tables) => {
+  const [ TableName, ...unprocessedTables ] = tables;
+  const timestamp = moment().startOf('week').unix() * 1000;
+  console.log('Scanning table - ', TableName);
+
+  const { Items } = await scan({
+    TableName,
+    FilterExpression: 'id > :timestamp',
```

Comment:
why to do filter by date? in the latest.csv I need info about all the weeks. For every week we need the latest records